CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v3.1.0
+ 1. Added full support to OneView Rest API version 300 for the hardware variants C7000 and Synergy to the already existing features:
+   - Uplink Set
+
```

Comment:
It's probably good to also add:

### Features supported
   - Connection template
   - Datacenter
   - Enclosure
   - Ethernet network
   - Fabrics
   - FC network
   - FCoE network
   - Firmware bundles
   - Firmware drivers
   - Logical downlink
   - Logical enclosure
   - Logical interconnect
   - Logical interconnect Group
   - Uplink Set
   - Logical switch
   - Logical switch group
   - Managed SANs
   - Network set
   - Power devices
   - Racks
   - SAN managers
   - Server hardware
   - Server hardware type
   - Server profile
   - Server profile template
   - Storage pools
   - Storage systems
   - Switches
   - Volume
   - Volume template
   - Drive Enclosures
   - Interconnect Link Topology
   - Internal Link Set
   - SAS Interconnect
   - SAS Interconnect Type
   - SAS Logical Interconnect
   - SAS Logical Interconnect Group
   - SAS Logical JBOD Attachments
   - SAS Logical JBODs

So that this list can contain everything that is supported by this full support release.